systemdev/calendars.rst

```diff
@@ -0,0 +1,54 @@
+Working on the Calendar feature
+===============================
+
+The Calendar feature is provided by several components:
+
+* The `Calendar App <https://gitlab.com/ubports/apps/calendar-app>`_, providing the user interface
+
+* `Evolution Data Server <https://github.com/ubports/evolution-data-server-packaging>`_ (often shortened as "EDS"), which is the backend where your calendars are stored
+
+* `sync-monitor <https://github.com/ubports/sync-monitor>`_, the service responsible for managing the synchronisation with remote calendars
+
+* `SyncEvolution <https://github.com/ubports/syncevolution-packaging>`_, the service responsible for performing the synchronisation to a WebDAV/CalDAV remote calendar
+
+
+Debugging
+---------
+
+Calendar synchronisation issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the device, it's possible to run ``syncevolution`` in debug mode. To do so, open a terminal (it's easier if done from a remote shell) and kill any existing processes:
```

Comment:
This is a good place to link the pages "Shell access via ssh" and "Shell access via adb", like what's done in the other systemdev pages.

```
You can gain a shell on the device using :doc:`/userguide/advanceduse/adb` or :doc:`/userguide/advanceduse/ssh`. 
```